iroha/docs/Cargo.toml

```diff
@@ -10,5 +10,5 @@ edition = "2018"
 iroha = { path = "../../iroha" }
 iroha_error = { path = "../../iroha_error" }
 iroha_config = { path = "../../iroha_config" }
-async-std = "1.6.2"
 serde_json = "1"
+tokio = { version = "1.6.0", features = ["rt", "rt-multi-thread", "macros"]}
```

Comment:
I guess it doesn't need to have tokio

Reply:
Yes, could be removed now that I converted all methods to sync.